Makefile

```diff
@@ -78,6 +78,8 @@ setup:
 build: build-magdeck build-tempdeck build-thermocycler
 
 DUMMY_BOARD := false
+USE_GCODES := true
+LID_WARNING := true
```

Comment:
Oh these should be assigned with [?=](https://www.gnu.org/software/make/manual/html_node/Setting.html), that way you can do `make build-rthermocycler USE_GCODES=false`